images/app-python/Dockerfile

```diff
@@ -28,6 +28,7 @@ RUN python3 -m pip install --upgrade pip
 # This is required for GPIO and UART access using FTDI
 RUN apt install -y libftdi1
 RUN python3 -m pip install pyserial
+RUN python3 -m pip install python-can
```

Comment:
Add a comment above this like the comment regarding GPIO/UART so it is clear this is for CAN bus interaction

Reply:
Yes. Added comment.